hudi-cli/src/main/java/org/apache/hudi/cli/commands/SparkMain.java

```diff
@@ -62,7 +63,9 @@ public static void main(String[] args) throws Exception {
 
     SparkCommand cmd = SparkCommand.valueOf(command);
 
-    JavaSparkContext jsc = SparkUtil.initJavaSparkConf("hoodie-cli-" + command);
+    JavaSparkContext jsc = cmd == SparkCommand.CLEAN
```

Comment:
Why do you want to implement it specifically for CLEAN command? Any specific reasons? I could not understand the purpose of this PR basically. :)

Reply:
@pratyakshsharma only CLEAN command can  specify the sparkMaster now. Otherwise, `sparkMaster` not contained in `args` of other command.

Reply:
I have the same concern with @pratyakshsharma , can we make other commands also support specifying the spark master?

Reply:
@yanghua yep, it is better to add sparkMaster  for other commands. Is it need in this PR? I think I can create a new jira to do this, and then we can init jsc by sparkMaster for all commands.

Reply:
@hddong I can see in SparkMain.java class that sparkMaster and sparkMemory are both present in args of other command as well. For example - 

1. https://github.com/apache/incubator-hudi/blob/04449f33feb300b99750c52ec37f2561aa644456/hudi-cli/src/main/java/org/apache/hudi/cli/commands/SparkMain.java#L216
2. https://github.com/apache/incubator-hudi/blob/04449f33feb300b99750c52ec37f2561aa644456/hudi-cli/src/main/java/org/apache/hudi/cli/commands/SparkMain.java#L234

Am I missing something here? 

Also all these positional arguments will be changed to proper config objects as per this PR (https://github.com/apache/incubator-hudi/pull/1174). You might want to take a look at this one.

Reply:
@prashantwason my mistake, thanks for your point out.

Reply:
Let us address other commands also in this PR then :)

Reply:
@pratyakshsharma yes, it's need, i'm doing. Your PR was Left behind for too long :)

Reply:
sounds good.